oops.min.js

```diff
@@ -0,0 +1,115 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-var */
+/* eslint-disable max-len */
+
+// eslint-disable-next-line require-jsdoc
+function Oops(a) {
```

Comment:
- [ ] Your code look's too messy, if you can please consider using a non minified code so the code reviewer can check your code more easily